platform.js

```diff
@@ -99,26 +99,35 @@ class App {
   }
 
   copyTemplate () {
-    return fs.copy(this.opts.dir, this.originalResourcesAppDir, {
-      filter: ignore.userIgnoreFilter(this.opts),
-      dereference: this.opts.derefSymlinks
-    }).then(() => hooks.promisifyHooks(this.opts.afterCopy, [
-      this.originalResourcesAppDir,
-      this.opts.electronVersion,
-      this.opts.platform,
-      this.opts.arch
-    ])).then(() => {
-      if (this.opts.prune) {
-        return hooks.promisifyHooks(this.opts.afterPrune, [
-          this.originalResourcesAppDir,
-          this.opts.electronVersion,
-          this.opts.platform,
-          this.opts.arch
-        ])
-      } else {
-        return true
-      }
-    })
+    const asarOptions = common.createAsarOpts(this.opts) || {}
```

Comment:
At this point `asarOptions` should be an instance variable, no need to calculate it twice.